### PR TITLE
Image Edges Detector

### LINKED ISSUE #711: Add a python script to detect the edges of the image
I will add a python script that will act as canny edge detector and detect the edges of the images. I will use opencv for detecting the edges. I am a participant in DWOC and want to contribute. @HarshCasper Kindly, add the labels and please assign this issue to me.

### PR DESCRIPTION
**Pull request by Dhanush2612**
Python script to detect the edges of the image.
Readme included

Fixes #711

### CODE REVIEW

**Comment by Dhanush2612** [comment]
@vybhav72954 
This is for the issue #711

**Comment by vybhav72954** [comment]
Just something I am wondering

Capture


This is what ourPR template looks like, why do you delete it every time?

**Comment by vybhav72954** [comment]
Again there is no linked Issue



> @vybhav72954
> This is for the issue #711

This is not for linking works

Your PR body should contain Fixes #711

**Comment by Dhanush2612** [comment]
@vybhav72954 
I am really sorry, I didn't know about this. 
Should I make any changes currently for this PR.

**Comment by vybhav72954** [comment]
> @vybhav72954 
> I am really sorry, I didn't know about this. 
> Should I make any changes currently for this PR?
> 

Nopes, nothing as of now, am yet to review it. But please ensure to properly link issue next time and try to use PR template, they are here to help you out only.

**Comment by Dhanush2612** [comment]
@vybhav72954 
Thank you

File: Python/Edges Detector/setup.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```
Comment (vybhav72954) [inline Python/Edges Detector/setup.sh:1]: Plagiarized from [here](https://github.com/mafshar/edge-detection/blob/master/setup.sh).

File: Python/Edges Detector/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# edge-detection
```
Comment (vybhav72954) [inline Python/Edges Detector/README.md:1]: Plagiarized from [here](https://github.com/mafshar/edge-detection/blob/master/setup.sh).

**Review by vybhav72954 — COMMENTED** [review]
Entire code, copied from [here](https://github.com/mafshar/edge-detection)

**Comment by vybhav72954** [comment]
2nd Plag Flag

**Pull request closed without merging**